js/apps/admin-ui/src/components/realm-selector/RealmSelector.tsx

```diff
@@ -132,12 +167,11 @@ export const RealmSelector = () => {
     >
       {filteredItems.map((item) => (
         <ContextSelectorItemLink
-          key={item.name}
-          to={toDashboard({ realm: item.name })}
+          key={item.realm.name}
+          to={toDashboard({ realm: item.realm.name })}
           onClick={() => setOpen(false)}
         >
-          <RealmText value={item.name} />{" "}
-          {item.used && <Label>{t("recent")}</Label>}
+          <RealmText {...item.realm} showIsRecent={item.used} />{" "}
```

Comment:
I think we might be able omit this white space that is being inserted.
```suggestion
          <RealmText {...item.realm} showIsRecent={item.used} />
```